p2p/discover/lookup.go

```diff
@@ -106,6 +107,12 @@ func (it *lookup) startQueries() bool {
 		it.tab.mutex.Lock()
 		closest := it.tab.closest(it.result.target, bucketSize, false)
 		it.tab.mutex.Unlock()
+		// Avoid finishing the lookup too quickly if table is empty. It'd be better to wait
+		// for the table to fill in this case, but there is no good mechanism for that
+		// yet.
+		if len(closest.entries) == 0 {
+			time.Sleep(1 * time.Second)
```

Comment:
After having sleeped for a whole second, wouldn't it make sense to check `it.tab.closest` once more before returning empty?

Reply:
Since we're now using `lookupIterator` exclusively, it'll just come around and trigger another lookup right after one finishes without any result.

Reply:
`1 * time.Second` -> `time.Second`?